Makefile

```diff
@@ -82,7 +82,7 @@ devel-install: dist/index.js
 
 # when building a distribution tarball, call webpack with a 'production' environment
 dist-gzip: NODE_ENV=production
-dist-gzip: clean all
+dist-gzip: clean $(NODE_MODULES_TEST) all
```

Comment:
No, please drop this. This got deliberately [dropped in starter-kit](https://github.com/cockpit-project/starter-kit/commit/e9635620c57b78) as it breaks RPM builds that have no  network. Current cockpit-podman master is rebased against current starter-kit, and `dist-gzip` does exactly what it should now.  So please just rebase your branch against current origin/master.

Also, please don't do commits with such commit messages. They should describe *what* changed in high-level terms, and also *why* the change was made (unless it's obvious, like typo fixes).